installing/installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc

```diff
@@ -11,7 +11,7 @@ The following procedure is partially automated and requires manual steps after t
 
 == Prerequisites
 * You have read the following documentation:
-** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview[Cluster lifecycle with multicluster engine operator overview].
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/about/cluster_mce_overview[Cluster lifecycle with multicluster engine operator overview].
```

Comment:
This one is 404'ing as well. I notice the `2.9` didn't get changed to `2.12` here tho? And it looks like might also just be the `about/` causing the problem:
```suggestion
** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.12/html/clusters/cluster_mce_overview[Cluster lifecycle with multicluster engine operator overview].
```

Reply:
I opened https://github.com/openshift/openshift-docs/pull/84645 to make this change. Thank you!